spec/dummy/db/schema.rb

```diff
@@ -29,12 +29,13 @@
   create_table "oauth_access_tokens", force: true do |t|
     t.integer  "resource_owner_id"
     t.integer  "application_id"
-    t.string   "token",             null: false
+    t.string   "token",                               null: false
     t.string   "refresh_token"
     t.integer  "expires_in"
     t.datetime "revoked_at"
-    t.datetime "created_at",        null: false
+    t.datetime "created_at",                          null: false
     t.string   "scopes"
+    t.string   "previous_refresh_token", default: "", null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.